packages/scss/src/components/timeline/mods.scss

```diff
@@ -1,6 +1,142 @@
-@mixin number($step: '.timeline-step', $title: '.timeline-step-title', $between: '.timeline-stepAddBetweenStep') {
-	counter-reset: timeline;
+@use 'structure' as s;
+@use '../../commons/utils/reset';
+@use '../../../../icons/src/mixins' as icon;
+
+@mixin small($title: s.$title) { 
+	#{$title} {
+		padding-left: 1.25rem;
+
+		&::before {
+			width: 0.5rem;
+			height: 0.5rem;
+		}
+	}
+}
+
+@mixin big($step: s.$step, $title: s.$title, $icon: s.$icon) {
+	#{$title} {
+		padding-left: 2.25rem;
+		min-width: var(--spacings-big);
+
+		&::before {
+			width: 1.5rem;
+			height: 1.5rem;
+		}
+	}
+
+	#{$step}:not([aria-current='step']):not([aria-current='step'] ~ #{$step}) #{$icon},
+	&.mod-addStep #{$icon} {
+		&::before {
+			left: var(--spacings-smaller);
+		}
+	}
+}
+
+@mixin vertical($step: s.$step, $title: s.$title, $icon: s.$icon, $description: s.$description, $between: s.$between) {
+	align-items: flex-start;
+	flex-direction: column;
+	
+	#{$step} {
+		margin-top: var(--spacings-smaller);
+		display: block;
+		position: relative;
+
+		//line
+		&:not(:last-child)::after {
+			content: '';
+			display: block;
+			height: auto;
+			width: 2px;
+			position: absolute;
+			top: calc(var(--spacings-big) + var(--spacings-smallest));
+			bottom: -0.75rem;
+			left: 0.45rem; // magic number
+			margin: 0;
+		}
+
+		&:not(:last-child)[aria-current='step'] {
+			&::after {
+				top: calc(var(--spacings-big) + var(--spacings-smallest));
+			}
+		}
+	}
+
+	#{$title} {
+		margin: var(--spacings-smaller) 0;
+		min-height: var(--spacings-standard);
+		min-width: 0;
+	}
+
+	&:not(.mod-small):not(.mod-big) {
+		#{$title} {
+			&:before {
+				left: 0;
+			}
+		}
+	}
+
+	#{$description} {
+		padding-left: var(--spacings-big);
+	}
+
+	&.mod-small {
```

Comment:
You should not have a mod class statement here.

Reply:
When mods are combined, I use a dedicated mixin which I call from the index with
```
&.mod-vertical {
	&.mod-small {
		@include verticalSmall
	}
}
```